### PR TITLE
2022 02 colheaders

### PR DESCRIPTION
**Pull request by xwatkins**
## Purpose
Add tooltips for column headers

## Approach
Create config file to host the definition. Wire in to the data-table component.

## Testing
none

## Checklist
- [x] My PR is scoped properly, and “does one thing only”
- [x] I have reviewed my own code
- [x] I have checked that linting checks pass and type safety is respected
- [x] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why

### CODE REVIEW

File: src/shared/components/results/__tests__/__snapshots__/ResultsData.spec.tsx.snap [hunk 1]
```diff
@@ -3968,7 +3968,11 @@ exports[`ResultsData component should render the table view 1`] = `
           <th
             class=""
           >
-            Entry
+            <span
+              aria-expanded="false"
+            >
```
Comment (aurel-l) [inline src/shared/components/results/__tests__/__snapshots__/ResultsData.spec.tsx.snap:3973]: Is this an artifact of the tooltip library?

Comment (xwatkins) [reply src/shared/components/results/__tests__/__snapshots__/ResultsData.spec.tsx.snap:3973]: It looks like it - https://github.com/tvkhoa/react-tippy#accessibility

**Review by aurel-l — APPROVED** [review]
Looks OK in this PR, but:
 - Any reason why you used fragments over plain strings when there is no "more" link? (e.g. tooltip for `Checksum`)
 - Clicking on the "more" link triggers the sorting, and doesn't follow the link. But I'm guessing this needs to be solved in Franklin rather than in this PR, right?

**Comment by xwatkins** [comment]
> Looks OK in this PR, but:
> 
> * Any reason why you used fragments over plain strings when there is no "more" link? (e.g. tooltip for `Checksum`)
That comes from Dan's script, and I had to copy/paste all of them manually... had a quick check at the beginning but missed that. I guess it doesn't really matter in the big scheme of things?

> * Clicking on the "more" link triggers the sorting, and doesn't follow the link. But I'm guessing this needs to be solved in Franklin rather than in this PR, right?
Yes, that reminds me I need to make a Jira.

**Comment by aurel-l** [comment]
> > Looks OK in this PR, but:
> > 
> > * Any reason why you used fragments over plain strings when there is no "more" link? (e.g. tooltip for `Checksum`)
> >   That comes from Dan's script, and I had to copy/paste all of them manually... had a quick check at the beginning but missed that. I guess it doesn't really matter in the big scheme of things?

No it doesn't, I was just wondering if at some point there was a type issue that got resolved. But I don't think it makes any difference.

> 
> > * Clicking on the "more" link triggers the sorting, and doesn't follow the link. But I'm guessing this needs to be solved in Franklin rather than in this PR, right?
> >   Yes, that reminds me I need to make a Jira.

OK, good that it is something that you had already noticed and that we are keeping track of

**Pull request merged**